README.md

```diff
@@ -1,372 +1,332 @@
-# IndexedDB Promised
-
-This is a tiny library that mirrors IndexedDB, but replaces the weird `IDBRequest` objects with promises, plus a couple of other small changes.
+# IndexedDB with usability.
+
+This is a tiny (~1.13k) library that mostly mirrors the IndexedDB API, but with small improvements that make a big difference to usability.
```

Comment:
Did the size increase (up from ~0.7k) happen because of the addition of convenience methods? If so, do you think it'd be worth also having a version without them (if feasible)?

If the lib is smaller than the previous version, it's probably not a huge deal, but it could be nice for those who don't use indexes much.

Reply:
It's smaller than the previous version, so I'm not that bothered.

I went back and forth on this, but the `db.get` methods are so commonly useful, I think they're worth it. They are however defined as a plugin, so it's easy to do a separate build if there's demand.